<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepartamentForm</class>
 <widget class="QDialog" name="DepartamentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="leCountryId">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDepartamentId">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDepartamentNanme">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dtDepartamentCreated">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dtDepartamentUpdated">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Insertar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
